src/resolve/resolved-manager.h

```diff
@@ -39,6 +39,7 @@ struct Manager {
         DnssecMode dnssec_mode;
         DnsOverTlsMode dns_over_tls_mode;
         DnsCacheMode enable_cache;
+        bool enable_caching_from_localhost;
```

Comment:
Not a matter, but `enable` prefix sounds redundant for boolean variable.

Reply:
Agree, fixed.